code/modules/virus2/effect.dm

```diff
@@ -13,8 +13,8 @@
 		// How damaging the virus is. Higher values are worse.
 
 	var/chance = 3
-		// Under normal conditions, the percentage chance per tick to activate. 
-	var/max_chance = 50	
+		// Under normal conditions, the percentage chance per tick to activate.
+	var/max_chance = 6
```

Comment:
This seems like a bit of a balance change, making it a maximum chance of 6% per tick rather than 50%?

It'd make all random event viruses comparatively benign to what they are right now